### DISCUSSION TITLE
有大佬实践过樱花代理websocker流量么

### DISCUSSION THREAD

**Discussion by bananaive** [natfrp/wiki#118] [使用问题]
如题,我的应用在通过https建立链接后会通过websocket通信,使用樱花无法正常使用

**Comment by Klosw** [comment]
会不会是 你配置的不对 ,或者是别的不对呀

**Comment by wodepig** [comment]
创建一个https的隧道, 添加自动https
<img width="786" height="1022" alt="image" src="https://github.com/user-attachments/assets/ad22917e-5fd0-416e-a96f-173b2182bd15" />
连接的时候用wss;//xxx, 这样就可以了

**Comment by Seele-Vollerei32** [comment]
参考 https://doc.natfrp.com/app/http.html 如果依旧无法访问，请提供软件与相关配置
